src/vs/workbench/api/browser/mainThreadTask.ts

```diff
@@ -460,9 +460,24 @@ export class MainThreadTask implements MainThreadTaskShape {
 						extension: value.extension
 					} as TaskSet;
 				});
+			},
+			resolveTask: (task: ConfiguringTask) => {
+				const dto = TaskDTO.from(task);
+
+				// Only named tasks can be resolved. (That should be the case, right?)
```

Comment:
As far as I can see, not having a name is fine. The important thing is that the task has a definition so that the task provider knows what to do with it.